Day-11-Quiz-App/quizBuilder.js

```diff
@@ -168,7 +197,7 @@ class QuizBuilder {
         questionText.className = "inputStyles questionText";
         choices.className = "inputStyles questionChoices";
         answer.className = "inputStyles questionAnswer";
-        error.className = "error"
+        error.className = "error"        
```

Comment:
Here unneeded whitespace was added.

Reply:
Done in https://github.com/sik-leanix/100-days-of-javascript/pull/5/commits/6e08ec485496313f8cb74cdba0ede9ddc3153baf